src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo.Node.cs

```diff
@@ -63,11 +62,11 @@ private string GetDebuggerDisplay()
                 => Name + ", " + ParentIndex;
         }
 
-        private readonly struct ParameterTypeInfo(string name, bool isComplex, bool isArray)
+        private readonly record struct ParameterTypeInfo(string name, bool isComplex, bool isArray)
```

Comment:
❓ Why is this one changing?

Reply:
➡️ This type isn't used as a dictionary key, but rather as a dictionary value in a type that happens to create a set from the values.